Project4/style.css

```
@@ -0,0 +1,637 @@
+    -webkit-transition: all 0.5s linear;
+    -moz-transition: all 0.5s linear;
+    -o-transition: all 0.5s linear;
+    -ms-transition: all 0.5s linear;
+    transition: all 0.5s linear;
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: alpha(opacity=0);
+    opacity: 0;
+}
+
+.overlay-effect p {
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: alpha(opacity=0);
+    opacity: 0;
+    -webkit-transform: scale(0);
+    -moz-transform: scale(0);
+    -o-transform: scale(0);
+    -ms-transform: scale(0);
+    transform: scale(0);
+    -webkit-transition: all 0.5s linear;
+    -moz-transition: all 0.5s linear;
+    -o-transition: all 0.5s linear;
+    -ms-transition: all 0.5s linear;
+    transition: all 0.5s linear;
+}
+
+.overlay-effect a.btn {
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: alpha(opacity=0);
+    opacity: 0;
+    -webkit-transform: scale(0);
+    -moz-transform: scale(0);
+    -o-transform: scale(0);
+    -ms-transform: scale(0);
+    transform: scale(0);
+    -webkit-transition: all 0.5s linear;
+    -moz-transition: all 0.5s linear;
+    -o-transition: all 0.5s linear;
+    -ms-transition: all 0.5s linear;
+    transition: all 0.5s linear;
+}
+
+.overlay-effect:hover img {
+    -webkit-transform: scale(10);
+    -moz-transform: scale(10);
+    -o-transform: scale(10);
+    -ms-transform: scale(10);
+    transform: scale(10);
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: alpha(opacity=0);
+    opacity: 0;
+}
+
+.overlay-effect:hover .mask {
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=100)";
+    filter: alpha(opacity=100);
+    opacity: 1;
+}
+
+.overlay-effect:hover h3,
+.overlay-effect:hover p,
+.overlay-effect:hover a.btn {
+    -webkit-transform: scale(1);
+    -moz-transform: scale(1);
+    -o-transform: scale(1);
+    -ms-transform: scale(1);
+    transform: scale(1);
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=100)";
+    filter: alpha(opacity=100);
+    opacity: 1;
+}
+
+@media (min-width: 900px) {
+
+    .overlay-item h3,
+    .overlay-item p {
+        font-size: 30px;
+    }
+}
+
+@media (min-width: 1280px) {
+    .speaker-details {
+        display: inline-block;
+        vertical-align: top;
+        width: 30%;
+    }
+
+    .overlay-item h3,
+    .overlay-item p {
+        font-size: 50px;
+
+    }
+}
+
+.demo {
+    text-align: center;
+    margin-top: 150px;
+
+}
+
+.mt {
+    margin-top: 20px;
+    font-size: 20px;
+    position: relative;
+}
+
+.medium {
+    width: 160px;
+}
+
+
+.diagonal {
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+    position: relative;
+    line-height: 50px;
+    background: #67809F;
+    color: white;
+    border: none;
+    font-weight: bold;
+    font-family: 'Open Sans',
+        sans-serif;
+    overflow: hidden;
+    z-index: 1;
+    padding: 0px;
```

Zero values don't need units:

```suggestion
    padding: 0;
```